doc/README

```diff
@@ -315,7 +320,7 @@ considered on the side of experimental.
 
 - Core
 
-  Packages: Hadoop, Spark, Hbase, Pig, Zookeeper
+  Packages: Hadoop, Spark, Hbase, Hive, Pig, Zookeeper
```

Comment:
too early to consider this part of the core of Magpie.  Lets add it to the experimental section :-)